### PR TITLE
documentation updates

### PR DESCRIPTION
**Pull request by antyona**
This PR is a draft for review. It will be merged after code freeze. 
The following issues are addressed by these changes: 
- Upgrade: https://github.ibm.com/IBMPrivateCloud/roadmap/issues/53122
- Ingress examples: https://github.ibm.com/IBMPrivateCloud/roadmap/issues/53121
- Additional steps when install LS on IKS\EKS : https://github.ibm.com/IBMPrivateCloud/roadmap/issues/53100
- What's New added for 1.15

@geforcelive Let's discuss the changes

### CODE REVIEW

File: docs/Content/Backup_and_upgrade.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ kubectl get csv -n "${licensingNamespace}" "${csv_name}" -o jsonpath='{.status.p
 
 ### Updating the location of the CatalogSource
 
-If you are upgrading from version 1.10.x or earlier, you must update the location of the `CatalogSource`.
+If you are upgrading from version 1.11.x or earlier, you must update the location of the `CatalogSource`.
```
Comment (geforcelive) [inline docs/Content/Backup_and_upgrade.md:62]: upgade ok

File: docs/Content/Configuration.md [hunk 2]
```diff
@@ -14,7 +14,9 @@ After you install License Service, you can configure License Service if needed.
 
 You can configure ingress, for example, by completing the following steps:
 
-1\. Get the nginx ingress controller. You might get it, for example, from here: [https://kubernetes.github.io/ingress-nginx/deploy](https://kubernetes.github.io/ingress-nginx/deploy).
+1\. Get the nginx ingress controller. You might get it, for example, from [https://kubernetes.github.io/ingress-nginx/deploy](https://kubernetes.github.io/ingress-nginx/deploy).
+
+**Note:** If you already have ingress controller on the cluster, you do not need to get another one.
```
Comment (geforcelive) [inline docs/Content/Configuration.md:19]: ingress ok

File: docs/Content/Automatic_installation.md [hunk 3]
```diff
@@ -27,8 +27,11 @@ This procedure guides you through the installation of License Service. It does n
 
 2\. Run the script.
 
-**Results:**
-Installation is complete and **License Service** is running in your cluster.
+**What's Next:**
+
+- If you are installing License Service on IBM Cloud Kubernetes Services (IKS) or Amazon Elastic Kubernetes Service (EKS), as the following step you need to configure ingress. For more information, see [Configuring ingress](Configuration.md##configuring-ingress). After yu do, verify the installation.
+
+- If you are installing License Service on OCP, the installation is is complete at this point and **License Service** is running in your cluster. You can verify the installation.
```
Comment (antyona) [inline docs/Content/Automatic_installation.md:35]: @martabereta Is this ok?

File: docs/Content/Install_without_OLM.md [hunk 4]
```diff
@@ -126,9 +126,11 @@ spec:
...
 ```
 
-**Results:**
-Give operator couple minutes to configure all needed components.
-Installation is complete and **License Service** is running in your cluster.
+**What's Next:**
+
+- If you are installing License Service on IBM Cloud Kubernetes Services (IKS) or Amazon Elastic Kubernetes Service (EKS), as the following step you need to configure ingress. For more information, see [Configuring ingress](Configuration.md##configuring-ingress). After yu do, verify the installation.
+
+- If you are installing License Service on OCP, the installation is is complete at this point and **License Service** is running in your cluster. You can verify the installation.
```
Comment (antyona) [inline docs/Content/Install_without_OLM.md:134]: @martabereta Is this ok?

**Review by SzymonKowalczyk — APPROVED** [review]

**Pull request merged**